src/analysis/processing/qgsalgorithmdeleteduplicategeometries.cpp

```diff
@@ -188,7 +188,7 @@ QVariantMap QgsDeleteDuplicateGeometriesAlgorithm::processAlgorithm( const QVari
     feedback->setProgress( 0.10 * current * step + 90 ); // takes about 10% of time
   }
 
-  feedback->pushInfo( QObject::tr( "%1 duplicate features removed" ).arg( removed ) );
+  feedback->pushInfo( QObject::tr( "%n duplicate features removed", nullptr, removed ) );
```

Comment:
```suggestion
  feedback->pushInfo( QObject::tr( "%n duplicate feature(s) removed", nullptr, removed ) );
```

Reply:
Now that we translate to en-US, I wonder if using `(s)` is really necessary. It makes translation in Transifex less straightforward (I went through two days ago), since you have to alter the two forms (removing `(`, `)`, `(s)`) while you could just copy it both and remove the s for the singular

Reply:
It's not necessary at all. It's just how it's reflected on dev builds.

Reply:
Thinking more about it... I think it will also fallback to these strings for incomplete translation (whatever language) in releases, so probably better to use the `(s)` form, more syntactically correct in most cases than e.g. `1 features`. And also rules would be the same for all, and follow the official syntax. And too bad for en-US translators 😢